client/src/components/App/AppToolbar/AppToolbarStyles.ts

```diff
@@ -53,6 +53,12 @@ const useStyles = makeStyles((theme: Theme) => ({
         color: 'white',
         padding: theme.spacing(1),
         marginLeft: theme.spacing(1)
+    },
+    countySelect: {
+        color: 'white',
+        '&:focus': {
+            opacity: 'unset'
```

Comment:
Why unset? What do you want the value of opacity to be?

Reply:
I wanted the opacity not to be shown at all when selecting on the select, that why I removed it by unset

Reply:
Can't you can just set the opacity to 1 or something? I think relying on parental styles is kind of unsafe